mysql-test/suite/rocksdb/t/flush_log_at_trx_commit.test

```diff
@@ -9,8 +9,8 @@ SET GLOBAL rocksdb_write_ignore_missing_column_families=true;
 
 create table aaa (id int primary key, i int) engine=rocksdb;
 
-SET GLOBAL rocksdb_flush_log_at_trx_commit=0;
---exec sleep 30
+SET GLOBAL rocksdb_flush_log_at_trx_commit=1;
+--exec sleep 5
```

Comment:
What is the purpose of this and other sleeps in this testcase in the first place?

Reply:
I believe the sleeps are to wait for the background timed flusher to wake/complete. Using debug variables or debug sync points would be ideal, but, the background flusher is down in rocksdb code which has no/difficult access to higher layer syncing tools.

Reply:
Not necessarily: if there were a status counter how many times a background task was executed, it wouldn't be that hard to access and would allow release build testing too